themes/survey/ls6_surveytheme/css/variations/theme_red.css

```diff
@@ -14572,6 +14572,37 @@ input[type=checkbox]:checked:after,
   width: 24px;
 }
 
+body.ls6_surveytheme {
```

Comment:
dont use ls6_surveytheme here.
1. class name is about to change
2. your adjustments for the body will be used globally in the survey.

You want to change the body for error page only, then add custom class for body in layout_error.twig
For example "error-page" and use it like that: body.error-page {...}